production/scrapers/youth_manual.R

```diff
@@ -0,0 +1,93 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+youth_manual_pull <- function(x){
+    manual_sheet <- googlesheets4::read_sheet(ss = "17mC-uHp1jhMQO8JGqn4is6pJLrKHP0G0TR57R01MxrY",
+                                              sheet = "Permanent", 
+                                              col_types = "c")
+    return(manual_sheet)
+}
+
+youth_manual_restruct <- function(x){
+    out <- x %>%
+        mutate(Date = lubridate::mdy(`Date of last positive case/last update`)) %>%
+        ## filter to dates within the last month, & no dates in the future 
+        filter(Date  >= lubridate::today() - lubridate::days(30),
+               Date <= lubridate::today()) %>%
+        mutate(Name = `County/Name of Facility`,
+               Name = str_glue('{Name} YOUTH FACILITY {State}'),
+               Residents.Confirmed = string_to_clean_numeric(`Confirmed Cases (Youth)`),
+               Residents.Active = string_to_clean_numeric(`Confirmed Cases (Youth)`),
+               Staff.Confirmed = string_to_clean_numeric(`Active Cases (Youth)`),
+               Staff.Deaths = string_to_clean_numeric(`Confirmed Deaths (Staff)`),
+               Staff.Active = string_to_clean_numeric(`Active Cases Staff`),
+               Residents.Population = string_to_clean_numeric(`Youth Population`)) %>%
+        relocate(any_of(starts_with("Residents.")),
+                 any_of(starts_with("Staff."))) %>%
+        rowwise() %>%
+        mutate(total_numeric = sum_na_rm(c_across(Residents.Confirmed:Staff.Active))) %>%
+        ## filter out rows where all numeric vars are NA 
+        filter(!is.na(total_numeric))
+    
+    return(out)
+}
+
+youth_manual_extract <- function(x){
+    x %>%
+        select(
+            Name,
+            Residents.Confirmed,
+            Residents.Active,
+            Residents.Population,
+            Staff.Confirmed,
+            Staff.Active,
+            Staff.Deaths
+        ) 
+}
+
+#' Scraper class for general youth_manual COVID data
+#' 
+#' @name youth_manual_scraper
+#' @description This will be a description of youth_manual data and what the scraper
+#' does
+#' \describe{
+#'   \item{Facility_Name}{The faciilty name.}
+#' }
+
+youth_manual_scraper <- R6Class(
+    "youth_manual_scraper",
+    inherit = generic_scraper,
+    public = list(
+        log = NULL,
+        initialize = function(
+            log,
+            url = "Youth team manual data collection",
```

Comment:
Super minor, but thoughts on making this just "Manual data collection" to be more intuitive for people outside our team who might not know what the Youth team refers to?